docs/README.md

```diff
@@ -1,14 +1,3 @@
 # Kubeapps documentation
 
-Everything you need to know about Kubeapps.
-
-## Documentation overview
-
-A high-level overview of how it’s organized will help Kubeapps contributors and users know where to look for certain things.
-
-| Section                     | Description                                                                              |
-| --------------------------- | ---------------------------------------------------------------------------------------- |
-| [Tutorials](./tutorials/)   | Start here if you’re new to Kubeapps. A hands-on introduction to Kubeapps for new users. |
-| [How-to guides](./howto/)   | They guide you through the steps involved in addressing key problems and use-cases.      |
-| [Background](./background/) | Dive into the overall architecture and implementation details of Kubeapps.               |
-| [Reference](./reference/)   | Technical information - developer guides, design proposals, examples, translations, etc. |
+The documentation files have moved to [/site/content/docs/latest](/site/content/docs/latest).
```

Comment:
Big downside... docs have to be under the hugo site subfolder in order to get them rendered :( That's why I'm moving the docks down there.
Good news is that we will get pretty URLs like: `/docs/latest/howto/oidc/azure-active-directory/` based on the md's name

Reply:
They can't be symlinked like you've done for other files? (Might not be the best solution, as then relative refs to other static content might be painful, just wondering).

Reply:
I've tried to create a symlink to the folder but it didn't work in GitHub: it rendered as a text file with a link to the symlinked folder :(  
However, instead of just saying "ey, we have moved it", we can leave the links to the howtos, background, refs, so that users can click on them and get redirected to the proper file. I'll do that.

Reply:
Oh - I actually meant whether it was possible the other way around - leave the docs where they are currently, and symlink to them from the hugo content source. But yep, guessing. you've tried all the options.

Reply:
And, I think it's fine having them in ./site/content/ anyway. Although, why are is the `docs/latest/` needed? Aren't the docs in our main branch always going to be the latest? (ie. why have you not used `site/content/tutorials/getting-started.md` rather than `site/content/docs/latest/tutorials/getting-started.md`? - I assume there's a good reason, just keen to understand). I assume the `latest` is so we can have multiple versions in our one main branch so hugo can render docs for multiple versions at once? And I assume the extra `docs` is so we can have `./site/content/static/` etc. as well. Just wondering if there might be a simpler structure to achieve those (particularly the first, whether we could have just one copy of the site in the branch - not sure).

Reply:
Yep, it would have been easier if it wasn't for the template. It is prepared to host several docs versions or, if there is only one, it gets served under `latest`. The template has some parameters for handling that and the whole template is prepared for that. I didn't want to diverge that much from the official template, so that's why I left the core structure.
Anyway, happy to change it in a follow-up PR.

Reply:
Oh - it's a constraint of the template... yeah, I agree, let's stick to the template :)